frontend/app/src/components/asset-manager/ManagedAssetTable.vue

```diff
@@ -274,11 +268,25 @@ const getAsset = (item: SupportedAsset) => {
   };
 };
 
+const { setMessage } = useMessageStore();
+const { isAssetIgnored, ignoreAsset, unignoreAsset } = useIgnoredAssetsStore();
+
+const isIgnored = (identifier: string) => {
+  return isAssetIgnored(identifier);
+};
+
 const toggleIgnoreAsset = async (identifier: string) => {
+  let success = false;
   if (get(isIgnored(identifier))) {
-    await unignoreAsset(get(identifier));
+    const response = await unignoreAsset(identifier);
+    success = response.success;
   } else {
-    await ignoreAsset(get(identifier));
+    const response = await ignoreAsset(identifier);
+    success = response.success;
+  }
+
+  if (success && get(ignoredAssetsHandling) !== 'none') {
+    updatePaginationHandler(get(options));
```

Comment:
At least from the review interface, it is not obvious what is the reason for the `!== 'none'` check here. Can you give me a bit more info on this?

Reply:
Okay, so the purpose here is to refresh the assets data, after we ignore / unignore an assets.
But we don't need to refresh it if `ignoreAssetsHandling==='none'`, because it still shows all data.
But if the filter is `excludes`, it should refresh the assets data, and remove the assets we just ignore.